Doc/library/ast.rst

```diff
@@ -1761,10 +1761,34 @@ Pattern matching
 
    .. versionadded:: 3.10
 
+Type annotations
+^^^^^^^^^^^^^^^^
+
+.. class:: TypeIgnore(lineno, tag)
+
+   A type ignore comment. ``lineno`` is the location of the ``#type ignore`` comment.
+   ``tag`` is the optional tag specified by the form ``#type ignore <tag>``.
+
+   see :func:`ast.parse` for more details.
+
+   .. doctest::
+
+      >>> print(ast.dump(ast.parse('x = 1 # type: ignore', type_comments=True), indent=4))
```

Comment:
Please, add a note that `TypeIgnore` nodes are not generated when `type_comments` parameter is `False` (default).